presto-main/src/test/java/io/prestosql/sql/query/TestSubqueries.java

```diff
@@ -79,7 +79,7 @@ public void testUnsupportedSubqueriesWithCoercions()
     {
         // coercion FROM subquery symbol type to correlation type
         assertions.assertFails(
-                "SELECT (SELECT count(*) FROM (VALUES 1) t(a) WHERE t.a=t2.b GROUP BY t.a LIMIT 1) FROM (VALUES 1.0) t2(b)",
+                "SELECT EXISTS(SELECT 1 FROM (VALUES (1, null)) t(a, b) WHERE t.a=t2.b GROUP BY t.b) FROM (VALUES 1.0, 2.0) t2(b)",
```

Comment:
Why a change here?

Reply:
limit is removed and because we estimate aggregation cardinality better, then we are left with scalar subquery for which we have more broad support.

I've modified test case so that it still tests that coercions are not supported within advanced subquery.